src_erl/Communication_Layer/http_Nerlserver/src/nerl_tools.hrl

```diff
@@ -27,6 +27,7 @@
 %% sorce tmpData name
 -define(TMP_DIRECTORY,"/tmp/nerlnet/").
 -define(TMP_DATA_ADDR, "tmpData.csv").
+-define(DATA_LEN, 1000*1000*1000). % default is 8MB, here set to 1000MB
```

Comment:
@halfway258  
What is this DATA_LEN? what kind of data?

Reply:
added comment